src/app/dashboard/dashboard.component.html

```diff
@@ -1,39 +1,18 @@
 <div class="container is-fluid">
-  <ng-progress [positionUsing]="'marginLeft'" [minimum]="0.15" [maximum]="1"
-    [speed]="200" [showSpinner]="true" [direction]="'leftToRightIncreased'"
-    [color]="'#1B95E0'" [trickleSpeed]="300" [thick]="true" [ease]="'linear'">
-  </ng-progress>
-  <div class="tile">
-    <div class="tile is-parent">
-      <article class="tile is-child box">
-        <h4> Readings Statistics History </h4>
-        <br/>
-        <chart [type]="readingChart" [data]="readingValues" [options]="chartOptions"></chart>
-        <br/>
-        <h6>Total since startup:<b> {{statisticsData['READINGS']}} </b></h6>
-      </article>
+    <ng-progress [positionUsing]="'marginLeft'" [minimum]="0.15" [maximum]="1"
+      [speed]="200" [showSpinner]="true" [direction]="'leftToRightIncreased'"
+      [color]="'#1B95E0'" [trickleSpeed]="300" [thick]="true" [ease]="'linear'">
+    </ng-progress>
+
+    <div class="tile" *ngFor="let statistic of statistics">
```

Comment:
can we filter (use filter from https://lodash.com/docs/4.17.4 itself) this at ViewModel(controller file) level?